2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,38 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let recipe = {
+  title: "rajma",
+  serving: 5,
+  ingredients: ["kidney beans", "spices", "tomato" ,"onion", "salt"],
+}
+let recipe1 = {
+  title: "pickle",
+  serving: 3,
+  ingredients: ["chilli", "garlic", "tomato" ,"onion", "salt"],
+}
+let recipe2 = {
+  title: "salad",
+  serving: 2,
+  ingredients: ["cabbage", "olives", "lemon sauce" ,"onion", "cucumber"],
+}
+let recipe3 = {
+  title: "daal",
+  serving: 5,
+  ingredients: ["lentil", "spices", "tomato" ,"onion", "salt"],
+}
+let recipe4 = {
+  title: "pasta",
+  serving: 5,
+  ingredients: ["pasta", "spices", "tomato" ,"cheese", "salt"],
+}
+
```

Comment:
The recipes you've created above look good 😄 
But have another look at the instructions on lines 13-20. Seems like they want you to output the recipe in this way:
```
Mole
Serves: 2
Ingredients:
cinnamon
cumin
cocoa
```